index.html

```diff
@@ -388,6 +388,28 @@
          as specified in [[!RFC7231]] <a href='https://tools.ietf.org/html/rfc7231#section-4.3.2'>section 4.3.2</a>.
         </p>
       </section>
+
+      <section id="httpDELETE">
+        <h2>HTTP DELETE</h2>
+        <p>
+         The <code>DELETE</code> method is optional per [[!LDP]] <a href='https://www.w3.org/TR/ldp/#ldpr-HTTP_DELETE'>section
+         4.2.5</a> and this specification does not require Fedora servers to implement it.  When a Fedora server supports this method,
+         in addition to the requirements imposed on <a>LDPR</a>s within containers outlined in [[!LDP]]
+         <a href='https://www.w3.org/TR/ldp/#ldpc-HTTP_DELETE'> section 5.2.5</a>, it must also follow the additional behavior outlined
+         below.
+        </p>
+        <section id="httpDELETEDepth">
+          <h3>Depth Header</h3>
+          <p>
+          A Feodra server MAY support the <code>Depth</code> header, as defined in [[!RFC4918]] <a href='https://tools.ietf.org/html/rfc4918#section-10.2'> section
+          10.2 </a>. Implementations MAY choose not to support all of the header's values and MUST reject a request containing an unsupported <code>Depth</code>
+          value with a 400 (Bad Request).  Implementations MUST define their own default behavior if a Depth header is not present.
+          </p>
+          <p>
+          The recursion of the <code>DELETE</code> MUST be reckoned along the <code>ldp:contains</code> axis linking contained resources.
```

Comment:
You have repeatedly claimed that 

> 1, only has clear implications to me in the case that, for a given resource, it happens to be functionally equivalent to infinity because there is only a single level of contained resources.

and therefrom you argue that

> we [should] only specify 0 and infinity, a restriction that is allowable in the defining spec for the Depth header

Both @acoburn and myself have given you clear answers as to why your assumption is not correct (https://github.com/fcrepo/fcrepo-specification/pull/126#discussion_r122027607), (https://github.com/fcrepo/fcrepo-specification/issues/135#issuecomment-308518854) and that therefore your suggested restriction is not a good choice. As a side note, "arbitrary depth recursion" does seem to me to be very much part of the discussion for this function, and I would be very happy to learn otherwise and to see it explicitly excluded from the spec.